pages/edit.js

```diff
@@ -0,0 +1,365 @@
+import Image from 'next/image';
+import styled from 'styled-components';
+import hook from '../public/hook.svg';
+import add from '../public/add-button.svg';
+import remove from '../public/delete-button.svg';
+import FormHeadline from '../components/EditFormComponents/FormHeadline';
+import EditBackgroundImage from '../components/EditFormComponents/EditBackgroundImage';
+import useLocalStorage from '../hooks/useLocalStorage';
+import { useRouter } from 'next/router';
+
+export function changeTimerChain(event, id, newTimerChain, setNewTimerChain) {
+  if (event.target.id === 'name') {
```

Comment:
Why are the functions declared outside of the page?
If you declare them inside the

 ```jsx
export default function EditTimerChainForm({timerChain, setTimerChain, setTimerPointer, countdownKey,  setCountdownKey}) {
```

you won't need to give them the parameters "newTimerChain, setNewTimerChain". You can just call the methods because they are available for the page

Reply:
I have refactored the `changeTimerChain` now.

I need an export for the handleAdd to test ist.